.travis.yml

```diff
@@ -2,42 +2,31 @@ dist: xenial
 env:
   global:
   - COMMIT=${TRAVIS_COMMIT::8}
+  - GO111MODULE=on
 
 before_install:
   - sudo add-apt-repository ppa:opensatelliteproject/ppa -y
   - sudo add-apt-repository ppa:opensatelliteproject/drivers -y
   - sudo add-apt-repository ppa:myriadrf/gnuradio -y
   - sudo apt-get -qq update
   - sudo apt-get install -y  limesuite libaec-dev libaec0 liblimesuite-dev libcorrect libsathelper libsoapysdr0.6 libairspy0 libsoapysdr-dev libairspy-dev git g++ cmake libsqlite3-dev libi2c-dev libusb-1.0-0-dev
-  - echo "Building RTLSDR"
-  - git clone https://github.com/librtlsdr/librtlsdr.git
-  - cd librtlsdr
-  - mkdir build && cd build
-  - cmake ..
-  - make -j10
-  - sudo make install
-  - sudo ldconfig
```

Comment:
You removed this build process, and this is nescessary for building it.

Reply:
Ok, i think I understand this flow now. I thought that scripts in ./travis-build were just a duplication, but they are for release.